src/wix/WixToolset.Converters/WixConverter.cs

```diff
@@ -1739,8 +1779,8 @@ private void ConvertRemotePayloadElement(XElement element)
 
             this.OnInformation(ConverterTestType.BurnHashAlgorithmChanged, element, "The hash algorithm for bundles changed from SHA1 to SHA512.");
 
-            this.RemoveAttributeIfPresent(element, "CertificatePublicKey", ConverterTestType.BundleSignatureValidationObsolete, "The {0} element contains obsolete '{1}' attribute. Signature validation is no longer supported. The attribute will be removed.");
-            this.RemoveAttributeIfPresent(element, "CertificateThumbprint", ConverterTestType.BundleSignatureValidationObsolete, "The {0} element contains obsolete '{1}' attribute. Signature validation is no longer supported. The attribute will be removed.");
+            this.RemoveAttributeIfPresent(element, "CertificatePublicKey", ConverterTestType.BundleSignatureVerificationObsolete, "The {0} element contains obsolete '{1}' attribute. Signature verification is no longer supported. The attribute will be removed.");
+            this.RemoveAttributeIfPresent(element, "CertificateThumbprint", ConverterTestType.BundleSignatureVerificationObsolete, "The {0} element contains obsolete '{1}' attribute. Signature verification is no longer supported. The attribute will be removed.");
```

Comment:
I guess this is unrelated to your changes but why are we doing this? Signature verification is supported, though not through Catalogs and the signature information is never harvested/gathered during build.

Reply:
Perhaps that is old code from the times when signature verification was removed but then did not get re-removed when the feature came back? I'm not sure, I was just fixing the obvious wrong word.

I'll pull this change out and put it in its own PR for discussion.